src/views/partials/example.njk

```diff
@@ -34,7 +34,7 @@
                 example in a new tab
             </a>
         </div>
-        <iframe title="{{ params.item + " example" }}" id="{{ componentLabel }}-iframe" class="block border-0 w-full {{panelHeight}}" src="{{ exampleURL }}" frameBorder="0" loading="lazy">
+        <iframe title="{{ params.item + " example" }}" id="{{ componentLabel }}-iframe" class="block border-0 w-full resize-y {{panelHeight}}" src="{{ exampleURL }}" frameBorder="0" loading="lazy">
```

Comment:
Maybe something to be looked at in another ticket, but I think implementing some kind of iframe resize library might be useful as well to set the size more accordingly on page load (likely would be a progressive enhancement feature for JS users)